lib/parseConfig/getFunctionParameter.js

```diff
@@ -0,0 +1,24 @@
+/*
+* Given a function parameter, return a function that can be run at runtime
+* which evaluates the function string, and errors if required.
+*/
+var when = require('when');
+var _ = require('lodash');
+
+module.exports = function (parameter, key) {
+	return function () {
+		try {
+			eval('var evaluatedFn = '+parameter.value);	
```

Comment:
Why are you doing `'var evaluatedFn ='` as part of the evaluation? Can you not do `var evaluatedFn = eval(parameter.value);`?